manager/src/main/java/io/gingersnapproject/configuration/ConfigurationListener.java

```diff
@@ -0,0 +1,50 @@
+package io.gingersnapproject.configuration;
+
+import java.util.Map;
+
+import javax.enterprise.event.Observes;
+
+import org.eclipse.microprofile.config.ConfigProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.gingersnapproject.k8s.RuleEvents;
+import io.smallrye.config.SmallRyeConfig;
+
+public class ConfigurationListener {
+   SmallRyeConfig smConfig = ConfigProvider.getConfig().unwrap(SmallRyeConfig.class);
+   Configuration config = smConfig.getConfigMapping(Configuration.class);
```

Comment:
Can we just make this a bean and do `@Inject Configuration config`?

Reply:
for any reason it doesn't inject anything this way

Reply:
Have you tried declaring the class as a Bean with `@ApplicationScoped`?

Reply:
yeah, even with the refactoring it doesn't get injected, there must be some trick around